app-rails/config/environments/production.rb

```diff
@@ -75,9 +75,13 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment).
-  # config.active_job.queue_adapter = :resque
-  # config.active_job.queue_name_prefix = "app_rails_production"
+  # Use "GoodJob" for Active Job backend - docs at https://github.com/bensheldon/good_job
+  # GoodJob was picked as a default because it uses Postgres (already a part of this template)
+  # and has good adoption and maintenence.
```

Comment:
This should get moved to an ADR and a link to that should be added here.

Reply:
Will do, thanks!